### DISCUSSION TITLE
系统目录be下的core.* 是什么 清理机制是什么

### DISCUSSION THREAD

**Discussion by Linchuan-Yimcen** [apache/doris#8771] [A - General / Q&A] [answered]
系统目录be下的core.* 是什么 清理机制是什么，我看还有好久以前的文件

**Comment by dataalive** [comment, marked as answer]
这个是BE发生一些异常的时候出的core dump，需要自己清理。可以反馈下是什么版本，给我们下core来定位下。

**Comment by Linchuan-Yimcen** [comment]
多谢多谢 我用的0.15.1-rc09  这回先清理了 下回需要定位的时候再麻烦你们 直接删除文件会影响正在运行的服务吗

**Reply by dataalive** [reply]
直接删除那个core.xxx文件没有影响。

**Reply by Linchuan-Yimcen** [reply]
ok ok 另外有的be节点同级目录 还有 ?mx? 这种带问号的文件 就4，5kb 大小 ，因为是线上环境 没敢删，这个是什么呢，可以直接删除吗

**Reply by dataalive** [reply]
这个是不符合预期的，可以联系我们看看。wechat：Windcloud35926237
